aws-android-sdk-mobile-client/src/androidTest/java/com/amazonaws/mobile/client/AWSMobileClientPersistenceWithRestartabilityTest.java

```diff
@@ -492,12 +492,12 @@ private void mockHostedUISignIn() throws JSONException {
                 "CognitoIdentityProviderCache",
                 true);
 
-        final Set<String> scopes = new HashSet<String>(Arrays.asList("profile", "openid", "email"));
+        final Set<String> scopes = new HashSet<String>(Arrays.asList("aws.cognito.signin.user.admin", "phone", "openid", "profile", "email"));
```

Comment:
are these values stored as enum somewhere? I feel like it would be better to use well-defined enum here if we have any defined, and if not we should create them.

Reply:
These are specific strings from the awsconfiguration file so I don't think an ENUM would make sense in this case.